CLIENT-SPECIFICATION.md

```diff
@@ -164,7 +164,7 @@ If a page cannot be found in _any_ platform, then it is RECOMMENDED that clients
 https://github.com/tldr-pages/tldr/issues/new?title=page%20request:%20{command_name}
 ```
 
-where `{command_name}` is the name of the command that was not found.
+where `{command_name}` is the name of the command that was not found. It is RECOMMENDED that clients exit with a non-zero exit code in addition to showing the above message.
```

Comment:
I think we should word this more strongly. What about this?
```suggestion
where `{command_name}` is the name of the command that was not found. Clients that have control over their exit code on the command line (i.e. clients that provide a CLI) MUST exit with a non-zero exit code in addition to showing the above message.
```

This helps to provide a consistent scripting interface (e.g. for tools that detect whether pages exist or not).

Reply:
I agree with making it mandatory, though I had made it just recommended as I'm unsure how many clients would need changing to follow this. If no one has objections, I'll apply this change tomorrow.

Reply:
Right - good point. Still, I think having a requirement is a good idea. Please can you bump the version of the spec too? This might be considered a 'breaking' change to the spec, since we require something new.

Reply:
Committed changes, and bumped the version of the specification at the bottom.